addons/web/static/tests/control_panel/control_panel_tests.js

```diff
@@ -69,6 +70,7 @@ odoo.define('web.control_panel_tests', function (require) {
                 cpStoreConfig: {
                     viewInfo: { arch, fields: this.fields },
```

Comment:
@Arcasias I've rebased the change over 13.3 but the test fail because here `arch` is used as-is without calling `fields_view_get` on it, so the `modifiers` attribute is not added.

In 13.0, I see that the `createControlPanel` was effectively calling [fields_view_get](https://github.com/odoo/odoo/blob/13.0/addons/web/static/tests/helpers/test_utils_create.js#L280-L288).

How should I approach this ? 

Note: I've tested with a real search view/action - the `viewInfo` and `arch` do get the `modifiers`.

Reply:
Indeed, that's a problem we didn't take into account when refactoring the test helpers. We should definitely process the arch in `createControlPanel` before sending it to the model.

Here's the way I would approach the issue:
```js
if (config.viewInfo && config.viewInfo.arch) {
    const { arch, fields } = config.viewInfo;
    const model = "__mockmodel__";
    const serverParams = {
        model,
        data: { [model]: { fields, records: [] } },
    };
    const mockServer = await testUtilsMock.addMockEnvironment(
        new Widget(),
        serverParams
    );
    config.viewInfo.arch = mockServer.fieldsViewGet({
        arch,
        fields,
        model,
        viewOptions: { context: config.actionContext }
    }).arch;
}

```
I haven't looked thoroughly at the `mock_server.js` so there could be easiest ways to process the arch. Feel free to use this or any other method to process the arch before declaring the model.